index.html

```diff
@@ -9,6 +9,32 @@
   <body class="page">
     <div class="page__container">
       <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+      <section class="contacts">
+        <h2 class="contacts__title">Остались вопросы?Свяжитесь с нами!</h2>
+        <ul class="contacts__list">
+          <li class="contacts__item">
+            <h3 class="contacts__heading" >Звоните:</h3>
+            <p class="contacts__access">+7(812) 555-66-66</p>
+          </li>
+          <li class="contacts__item contacts__item_type_adress">
+            <h3 class="contacts__heading" >Приезжайте в гости:</h3>
+            <p class="contacts__access">ул.Большая Конюшенная, 19/8</p>
+          </li>
+          <li class="contacts__item contacts__item_type_email">
+            <h3 class="contacts__heading" >Пишите:</h3>
+            <p class="contacts__access">mail@htmlacademy.ru</p>
+          </li>
+        </ul>
+        <div class="contacts__map">
```

Comment:
@naumch1k  I did it.When you get a chance, could you please double-check to make sure I've done everything?

Reply:
@SergeyAlexandrowich just looked at the code. It actually looks good! 

Please double check if map size (height) match the design for all screen sizes. 

Also, didn't you forget white triangle over the map? **HINT**: instead of adding it as an `<img>`, you can try to set it as a background for map block pseudoelement :)

Let me know if you need any help or clarifications on that

Reply:
@naumch1k 
Third point: done
First point: set a min-height for all screen sizes

Reply:
@naumch1k  If you get the chance, could you please dive deeper into the second point for me?

Reply:
@SergeyAlexandrowich Absolutely! Please take a look at code below:

```CSS
.map {
  /* Use relative positioning for the map block to determine the position of our pseudo-element in relation to it */
  position: relative;
}

/* Styles for the pseudo-element ::before of the map block */
.map::before {

  /* Add an empty content to the pseudo-element for it to be generated and styled */
  content: "";

  /* Position pseudo-element */
  position: absolute;
  top: 0;
  left: 0;
  
  /* Define sizing of pseudo-element */
  width: 100%;
  height: 123px;
  
  /* pseudo-element background settings */
  background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1439 123'%3E%3Cpolygon points='0,0 1439,0 720,123' fill='white'/%3E%3C/svg%3E");
  background-repeat:  no-repeat;
  background-size: 100% auto;
  background-position: top center;
}
```

The design for this **_triangle mask_** is a little bit tricky and will take a lot of time and code to replicate the provided design. The primary goal here is to practice implementing it as a pseudo-element. 

You can read more on pseudo-elements [here](https://doka.guide/css/before/). 

Please, do not spend too much time trying to match the design; just ensure it is responsive and looks good when resizing the screen. (I think the provided code should be enough.)

Reply:
@naumch1k  I did it. However now the triangle is little overlapps the rectangle in the top-left corner
![map](https://github.com/p-foundation/pink-app/assets/121849648/51987a1b-5309-46a3-b861-2e318da66c5f)

Reply:
@SergeyAlexandrowich that's alright. We cannot really customize this map since we are using free option. Just live it like this. 

You did a great job!

Reply:
@SergeyAlexandrowich the last little thing I will ask you to address, is that

_We also should keep in mind, that [Nu Html Checker](https://validator.w3.org/nu/) will throw a warning if the section does not have a heading element. Since you don't have one per design file, you could still set one and hide it using a BEM mix `visually-hidden`. Check out this [thread](https://discord.com/channels/1062132018068070492/1162048644548145232/1168742224528425071) for details._

Reply:
@naumch1k I added heading element to section map